### PR TITLE
Ensure login shell is set correctly for Analytics users

### PR DESCRIPTION
**Pull request by itsjeyd**
This PR follows up on https://github.com/edx/configuration/pull/2753 and makes sure that the default login shell for the `analytics_api` and `insights` users is set to `/bin/bash` during provisioning.

**Test instructions**
1. Create a new base box for analyticstack by doing:
   
   ``` sh
   mkdir analytics
   cd analytics
   git clone https://github.com/open-craft/configuration.git
   cd configuration
   git checkout master-b3a7dc0
   virtualenv -p /usr/bin/python2 venv
   source venv/bin/activate
   pip install -r pre-requirements.txt
   pip install -r requirements.txt
   cd vagrant/base/analyticstack
   export OPENEDX_RELEASE=named-release/dogwood
   vagrant up
   ```
2. SSH into the VM and check the default shell for `analytics_api` and `insights` users:
   
   ``` sh
   vagrant ssh
   getent passwd analytics_api | cut -d: -f7
   getent passwd insights | cut -d: -f7
   ```
   
   The last two commands should output `/bin/bash`.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @itsjeyd! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fconfiguration&number=2834

File: playbooks/roles/analytics_api/meta/main.yml [hunk 1]
```diff
@@ -31,3 +31,11 @@ dependencies:
     edx_service_packages:
       debian: "{{ analytics_api_debian_pkgs }}"
       redhat: "{{ analytics_api_redhat_pkgs }}"
+  - role: local_dev
+    localdev_accounts:
```
Comment (feanil) [inline playbooks/roles/analytics_api/meta/main.yml]: Local Dev should never be a dependency of a role.  If we add it here it will also be added in production as well and this would be incorrect.  It should instead be added to the relevant playbooks.

File: playbooks/roles/insights/meta/main.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@
 #
 ##
 # Role includes for role insights
-# 
+#
```
Comment (mulby) [inline playbooks/roles/insights/meta/main.yml]: I would revert the changes to this file

**Comment by mulby** [comment]
one nit

:+1:

File: playbooks/roles/analytics_api/meta/main.yml (hunk 1 above)
Comment (itsjeyd) [inline playbooks/roles/analytics_api/meta/main.yml]: @feanil Thanks, I changed the approach based on your comment (and also addressed Gabe's nit below). Could you have another look please?

**Comment by feanil** [comment]
:+1:

**Comment by e-kolpakov** [comment]
@itsjeyd :+1:
